web/api/js/codechecker_api_js/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "codechecker_api_js",
+  "version": "6.24.0",
+  "description": "CodeChecker server javascript api stub module",
+  "main": "lib/codechecker_api_js",
+  "homepage": "https://github.com/Ericsson/codechecker",
+  "bugs": {
+    "url": "https://github.com/Ericsson/codechecker/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Ericsson/codechecker/tree/master/web/api"
+  },
+  "license": "Apache-2.0",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "codechecker",
+    "thrift",
+    "api"
+  ],
+  "dependencies": {
+    "thrift": "0.9.2"
```

Comment:
Cat it be greater than this version or do we need exactly this version of thrift?
We can specify it by using ~, ^, => etc. [See](https://docs.npmjs.com/misc/semver#ranges).